tools/make_old_docs_symlinks.py

```diff
@@ -0,0 +1,84 @@
+# Make redirects for our old html files relative to the root to version specific content.
+# See also ../docs/manual/src/README.md
+#
+# This is a "migration" script, so not run regularly and not by CI,
```

Comment:
Does that mean this can go away once we did the full migration?

Reply:
As per the comment, it might make sense to keep it around so the top-level html redirects can be bumped to a later version. If we think we never ever want to move them from 0.27 it could be removed. In fact it could be removed now, because I've already run it, but am yet to push it to gh-pages.